### PR TITLE
Convert `os initialize` to oclif, typescript

### PR DESCRIPTION
**Pull request by srlowe**
Change-type: patch
Signed-off-by: Scott Lowe <scott@balena.io>

### CODE REVIEW

**Comment by srlowe** [comment]
@balena-ci retest

File: lib/actions-oclif/os/initialize.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	public static authenticated = true;
+
+	public async run() {
+		const { args: params, flags: options } = this.parse<FlagsDef, ArgsDef>(
+			OsInitializeCmd,
+		);
+
+		const { promisify } = await import('bluebird');
+		const { umount } = await import('umount');
+		const umountAsync = promisify(umount);
```
Comment (srlowe) [inline lib/actions-oclif/os/initialize.ts:84]: I wasn't able to get `util.promisify` working with `umount` (ts complained about wrong number of parameters).

**Comment by srlowe** [comment]
@balena-ci retest

File: lib/actions-oclif/os/initialize.ts (hunk 1 above)
Comment (Page-) [reply lib/actions-oclif/os/initialize.ts:84]: It'll work if you change typings/umount/index.d.ts to
```ts
declare module 'umount' {
	export const umount: (
		device: string,
		callback: (err: Error | null, stdout?: any, stderr?: any) => void,
	) => void;
	export const isMounted: (
		device: string,
		callback: (err: Error | null, isMounted?: boolean) => void,
	) => void;
}

Comment (Page-) [reply lib/actions-oclif/os/initialize.ts:84]: Specifically `util.promisify` needs the first arg typing to be `Error | null`

Comment (srlowe) [reply lib/actions-oclif/os/initialize.ts:84]: Ah thanks!

**Pull request merged**